test/extended/builds/run_fs_verification.go

```diff
@@ -0,0 +1,136 @@
+package builds
+
+import (
+	"fmt"
+	"strings"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+)
+
+var _ = g.Describe("[sig-builds][Feature:Builds] verify /run filesystem contents", func() {
+	defer g.GinkgoRecover()
+	var (
+		oc                                      = exutil.NewCLI("verify-run-fs")
+		testVerityRunFSWriteableBuildConfigYaml = fmt.Sprintf(`
+apiVersion: build.openshift.io/v1
+kind: BuildConfig
+metadata:
+  name: verify-run-fs
+spec:
+  runPolicy: Serial
+  source:
+    dockerfile: "FROM %s\nRUN chmod -R uga+rwx /run\nUSER 1001"
```

Comment:
nit - can we use multiline YAML syntax?
```suggestion
    dockerfile: |
      FROM %s
      RUN chmod -R uga+rwx /run
      USER 1001"
```